mods/cnc/maps/nod04b/nod04b.lua

```diff
@@ -0,0 +1,200 @@
+	tran.Move(EntryPointVehicle.Location)
+end
+
+InsertNodUnits = function()
+	Reinforcements.ReinforceWithTransport(Nod, 'tran', NodUnitsBuggy, { EntryPointVehicle.Location, RallyPointVehicle.Location }, { EntryPointVehicle.Location }, nil, ChangeOwner)
+	Reinforcements.ReinforceWithTransport(Nod, 'tran', NodUnitsRocket, { EntryPointRocket.Location, RallyPointRocket.Location }, { EntryPointRocket.Location }, nil, ChangeOwner)
+	Reinforcements.ReinforceWithTransport(Nod, 'tran', NodUnitsGunner, { EntryPointGunner.Location, RallyPointGunner.Location }, { EntryPointGunner.Location }, nil, ChangeOwner)
+end
+
+WorldLoaded = function()
+	GDISupporter = Player.GetPlayer("GDISupporter")
+	NodReinforcements = Player.GetPlayer("NodReinforcements")
+	GDI = Player.GetPlayer("GDI")
+	Nod = Player.GetPlayer("Nod")
+	
+	Trigger.AfterDelay(Apc1TriggerFunctionTime, Apc1TriggerFunction)
+
+	Trigger.OnEnteredFootprint(Civ2CellTriggerActivator, function(a, id)
+		if a.Owner == Nod then
+			MyActors = getActors(GDI, { ['c6'] = 1, ['c7'] = 1, ['c8'] = 1, ['c9'] = 1 })
+			Utils.Do(MyActors, function(actor)
+				Civ2Movement(actor)
+			end)
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+	Trigger.OnEnteredFootprint(Civ1CellTriggerActivator, function(a, id)
+		if a.Owner == Nod then
+			MyActors = getActors(GDI, { ['c2'] = 1, ['c3'] = 1, ['c4'] = 1, ['c5'] = 1 })
+			Utils.Do(MyActors, function(actor)
+				Civ1Movement(actor)
+			end)
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+	
+	Trigger.OnPlayerDiscovered(GDISupporter, Hum1TriggerFunction)
+
+	Trigger.OnAllRemovedFromWorld(Apc2ActorTriggerActivator, Apc2TriggerFunction)
+
+	Trigger.OnEnteredFootprint(Apc3CellTriggerActivator, function(a, id)
+		if a.Owner == Nod then
+			MyActors = getActors(GDI, { ['apc'] = 1 })
+			Utils.Do(MyActors, function(actor)
+				Apc3Movement(actor)
+			end)
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+	Trigger.OnAllRemovedFromWorld(WinActorTriggerActivator, WinTriggerFunction)
+
+	Trigger.OnObjectiveAdded(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	Trigger.OnObjectiveCompleted(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(Nod, function()
+		Media.PlaySpeechNotification(Nod, "Win")
+	end)
+
+	Trigger.OnPlayerLost(Nod, function()
+		Media.PlaySpeechNotification(Nod, "Lose")
+	end)
+
+	NodObjective = GDI.AddPrimaryObjective("Kill all enemies!")
+	GDIObjective = Nod.AddPrimaryObjective("Destroy the village and kill all civilians!")
+
+	InsertNodUnits()
+end
+
+Tick = function()
+	if Nod.HasNoRequiredUnits() then
+		if DateTime.GameTime > 2 then
+			GDI.MarkCompletedObjective(GDIObjective)
+		end
+	end
+end
+
+getActors = function(owner, units)
```

Comment:
Please replace this with `GetActorsByType` and remove this just like you did in https://github.com/OpenRA/OpenRA/pull/7534/.